internal/providers/terraform/ibm/testdata/en_subscription_sms_test/en_subscription_sms_test.golden

```diff
@@ -0,0 +1,25 @@
+
+ Name                                                                  Monthly Qty  Unit                      Monthly Cost 
+                                                                                                                           
+ ibm_en_subscription_sms.subscription_sms_lite                                                                             
+ ├─ Outbound IBM Cloud SMS Message Units (Lite plan) (Max. 20)                  20  Message Units                    $0.00 
+ └─ SMS Number Setup Resource Units (Lite plan)                                  1  Resource Units                   $0.00 
+                                                                                                                           
+ ibm_en_subscription_sms.subscription_sms_standard                                                                         
+ ├─ Outbound IBM Cloud SMS Message Units (Standard plan)                     1,000  Message Units                   $16.15 
+ └─ SMS Number Setup Resource Units                                              1  Resource Units                   $1.00 
+                                                                                                                           
+ ibm_en_subscription_sms.subscription_sms_standard_no_usage                                                                
+ ├─ Outbound IBM Cloud SMS Message Units (Standard plan)        Monthly cost depends on usage: $0.01615 per Message Units  
+ └─ SMS Number Setup Resource Units                                              1  Resource Units                   $1.00 
```

Comment:
i thought these would be noted as a one-time fee?

Reply:
Good catch, I forgot to add that note — updated